locale/fa/download/releases.md

```diff
@@ -9,16 +9,12 @@ modules: "<code>NODE_MODULE_VERSION</code> refers to the ABI (application binary
  از آن جایی که آن‌ها بخشی از io.js fork بودند
   از Node.js 4.0.0 خط های انتشارهای قبلی  io.js در   Node.js 0.12.x  کاور شدند که به شکل انتشارهای یک پارجه در آمدند.
 
-<div class="highlight-box">
-    به دنبال آخرین انتشار از شاخه ورژن می‌گردید؟
-
-     <ul class="list-divider-pipe">
-        <li><a href="https://nodejs.org/dist/latest-v10.x/">Node.js 10.x</a></li>
-        <li><a href="https://nodejs.org/dist/latest-v8.x/">Node.js 8.x</a></li>
-        <li><a href="https://nodejs.org/dist/latest-v6.x/">Node.js 6.x</a></li>
-        <li><a href="https://nodejs.org/dist/latest-v4.x/">Node.js 4.x</a></li>
-        <li><a href="https://nodejs.org/dist/latest-v0.12.x/">Node.js 0.12.x</a></li>
-        <li><a href="https://nodejs.org/dist/latest-v0.10.x/">Node.js 0.10.x</a></li>
-        <li><a href="https://nodejs.org/dist/">تمامی نسخه‌ها</a></li>
-    </ul>
-</div>
+###  به دنبال آخرین انتشار از شاخه ورژن می‌گردید؟
```

Comment:
Please use the HTML provided on `master`.

Reply:
replaced.